### ISSUE TITLE
Underline Text in List

### ISSUE THREAD

**Issue by Michael-Grover** [accordproject/web-components#27]
The underline style can not be applied to text in an ordered or unordered list. The bold, italic, and code styles work for text in a list.

Example of a real contract where parts of an ordered list are underlined, which currently isn't possible in our editor:
![image](https://user-images.githubusercontent.com/40569793/63716087-2a3bb280-c813-11e9-904d-751eaf682765.png)

**Comment by K-Kumar-01** [comment]
@irmerk 
Can we use `<ins>` HTML tag for this purpose. I used it on template-dingus and it worked there
![Screenshot from 2021-03-03 00-44-11](https://user-images.githubusercontent.com/59891164/109701966-a4dc4f80-7bb9-11eb-9602-4d2d076b717a.png)
Though I am not sure about this but as we will wrap the text in HTML I don't think it would cause any issue as it is supported by markdown itself.

**Comment by jeromesimeon** [comment]
We shouldn't depend too much on HTML tags since we convert contracts to other formats: e.g., pdf, docx. It would be better to use a proper underline extension as @dselman suggested in a previous comment.

For instance: https://www.npmjs.com/package/markdown-it-plugin-underline

Not sure underline is our most pressing requirement though.

**Comment by K-Kumar-01** [comment]
@jeromesimeon @irmerk 
Since the priority is not pressing I would have a go at other issues then.

**Comment by Hanaffi** [comment]
@irmerk Can you assign this to me please?

**Comment by jeromesimeon** [comment]
> @irmerk Can you assign this to me please?

As you can see from the thread. We're not really pursuing this feature at the moment.

**Comment by Hanaffi** [comment]
@jeromesimeon Oh no.
Can you assign some issue to me? (For ex. https://github.com/accordproject/web-components/issues/73)
Every time I find an issue, Its assigned to someone or has low priority.

**Comment by irmerk** [comment]
If there aren't any open Issues that are open and not low priority, then there aren't any. Something as challenging as #73 won't be assigned to someone without having a pretty thorough investigation into what we need to do and how.

**Comment by Michael-Grover** [comment]
This issue is out of date. The tech working group is currently looking into adding underlined text to the markdown editor.

**Comment by irmerk** [comment]
Related to (blocked by if we want to pursue this) https://github.com/accordproject/markdown-transform/issues/368
